_layouts/default.html

```diff
@@ -39,6 +39,11 @@
     url: https://github.com/jupyter/
   - title: Mastodon
     url: https://hachyderm.io/@ProjectJupyter
+legal:
+  - title: Legal
```

Comment:
Yeahm you are right.,
```suggestion
  - title: Privacy
```